mpr-research-data.py

```diff
@@ -7,65 +7,34 @@
 import sqlalchemy as sql
 from google.cloud import storage
 
-logging.basicConfig(
-    format="%(asctime)s %(levelname)-8s [%(filename)s:%(lineno)d] - %(message)s",
-    datefmt="%Y-%m-%dT%H:%M:%S%z",
-    level=logging.INFO
-)
-
-# SETUP CONFIG VARIABLES
-# --------------------------------------------------------------------------
-targetBucketName = os.getenv('GCLOUD_BUCKET', 'mpr-research-data-uploads')
-
-numberOfMonths = os.getenv('NUMBER_OF_MONTHS', 4)
-
-defaultQueryFolder = os.getenv('QUERY_FOLDER', 'queries')
-queryTemplateDict = {'course': os.getenv('COURSE_QEURY', 'courseQuery.sql'),
-                     'retrieve': os.getenv('RETRIEVE_QUERY', 'retrieveQuery.sql')}
-
 
 # FUNCTIONS
 # --------------------------------------------------------------------------
 
-def getDBCreds():
-
-    dbCredsDefaultDict = {'NAME': None,
-                          'USER': None,
-                          'PASSWORD': None,
-                          'HOST': None,
-                          'PORT': 3306}
-    dbCredsDict = {}
+def setUpLogger():
 
-    allKeyPartsFound = True
-    for credPart in dbCredsDefaultDict:
-        dbCredsDict[credPart] = os.getenv(
-            'DB_' + credPart, dbCredsDefaultDict[credPart])
-
-        if not dbCredsDict[credPart]:
-            logging.error(
-                f'Did not find .env variable for M-Write Peer Review production DB key: {credPart}')
-            allKeyPartsFound = False
-
-    if not allKeyPartsFound:
-        sys.exit('Exiting.')
-
-    return dbCredsDict
-
-
-def getGCPCreds() -> dict:
-    gcpCredsDict = json.loads(os.getenv('GCP_KEY'))
-    return gcpCredsDict
+    logWarning = False
+    raiseLogTypeError = None
 
+    try:
+        logLevel = str(os.environ.get('LOG_LEVEL', 'DEBUG')).upper()
+    except TypeError as e:
+        logWarning = True
+        raiseLogTypeError = e
+    
+    if logLevel not in ['DEBUG', 'INFO', 'WARNING', 'ERROR', 'FATAL', 'CRITCAL', 'EXCEPTION']:
+        logWarning = True
 
-def queryRetriever(queryName, queryModifier=False,
```

Comment:
IMNSHO, it appears GH isn't doing a good job with diffs.   Showing lines of the other function which had been more than 30 lines later mixed in with lines of this doesn't make the diff very easy to read, does it?

(Just an observation.  There's nothing we can do about it.)